invenio/modules/tags/forms.py

```diff
@@ -68,15 +55,15 @@ def validate_tag_name(dummy_form, field):
                 _('The name must contain valid characters.'))
 
         if len(suggested_silent) > max_len:
-            raise validators.ValidationError( _('The name cannot exeed ')
-                    + str(max_len) + _(' characters.'))
+            raise validators.ValidationError(_('The name cannot exeed ')
```

Comment:
can you improve it to: `_('The name can not exceed %(x_max_len)d characters.', x_max_len=max_len)` ?